tests/e2e/Services/Account/AccountBase.php

```diff
@@ -1000,6 +1000,11 @@ public function testCreateAccountRecovery($data):array
 
         $recovery = substr($lastEmail['text'], strpos($lastEmail['text'], '&secret=', 0) + 8, 256);
 
+        $expireTime = strpos($lastEmail['text'], '&expires='.$response['body']['expire'], 0);
+
+        $this->assertNotFalse($expireTime);
```

Comment:
This test is perfect, but I can see we are missing tests for the other query params. Mind adding them?

Also please do the same with the verification tests 🙇🏻